### ISSUE TITLE
Update publish action

### ISSUE THREAD

**Issue by coret** [netwerk-digitaal-erfgoed/faq-for-iiif-and-lod#1]
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

@ddeboer can you help with issue?

**Comment by ddeboer** [comment]
Sure, but note that the actual build error is

    FATAL ERROR: Line 39 isn't indented enough (needs 1 indent) to be valid Markdown:
"*In contrast, PDFs are self-contained documents that do not readily allow for easy integration or comparison with other resources.*"
 ✘  Did not generate, due to fatal errors

**Comment by ddeboer** [comment]
Deprecation fixed in 4e33a4a09015280c43880ca706f218e4dd5178bc, but [note the build still fails](https://github.com/netwerk-digitaal-erfgoed/faq-for-iiif-and-lod/actions/runs/8085820877/job/22094199674).
